crates/astria-sequencer/src/ibc/component.rs

```diff
@@ -45,7 +45,9 @@ impl Component for IbcComponent {
             .wrap_err("failed to set IBC sudo key")?;
 
         for address in app_state.ibc_relayer_addresses() {
-            state.put_ibc_relayer_address(address);
+            state
+                .put_ibc_relayer_address(address)
+                .wrap_err("failed to put IBC relayer address")?;
```

Comment:
```suggestion
                .wrap_err("failed to write IBC relayer address to state")?;
```

Reply:
Done in [dd3bfa6](https://github.com/astriaorg/astria/pull/1492/commits/dd3bfa68494d3d14555a09c0dfb5cbe6228b6f7d).